### ISSUE TITLE
could you possibly update to include the stereo models

### ISSUE THREAD

**Issue by Dxtrmst** [facebookresearch/audiocraft#351]
could you possibly update to include the stereo models?
Thanks

**Comment by Ziegabeez** [comment]
Yeah I was excited to try upgrading from mono to stereo, but I'm extremely confused about how to make it work. I have no idea if the problem is that I'm too stupid to understand coding, or if the code itself isn't set up to use the stereo-large model. 

When MusicGen was first released, there was a great Google Colab that had all the code laid out (ie: not the currently available "Gradio" versions of Colab which are completely impractical to use since they only generate 1 clip at a time, and I can't figure out how to deconstruct that code). Could you please just release a straight forward example Colab that shows us how to: A) load all the necessary stuff onto the GPU, B) connect it to Google Drive for automatically saving the clips, C) a loop that generates X number of clips for each in a list of prompts, automatically saving them into an appropriate folder in the Google Drive. Also, D) if the model really does "continuation" then include that too with an explanation of what it does. Should include code to ensure no clobbering of previously generated clips. I was able to create a Colab that does all that with the mono model, but I want to use stereo.
